.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: python
+install:
+  - pip install -r data-analytics/dataflow-python-examples/requirements.txt
+  - pip install -e data-analytics/dataflow-python-examples/
+script:
+- python data-analytics/dataflow-python-examples/tests/test_parsing_logic.py
```

Comment:
Woah, hold up there. You're overriding the repository-wide CB file just to run a single test script for a single test in a single folder. Hold onto that thought, but remove this from this particular pull request.

Reply:
This seems like the only option :( https://github.com/travis-ci/travis-ci/issues/3540.  If we don't do this, our travis CI build fails.

Reply:
Nobody is relying on the "build" passing right now; Travis has been configured to work with a Ruby rake file since it was set up for this repo (and none of the code is in Ruby).

The value of a continuous build here is also highly questionable; your example code is highly isolated (nothing else depends on it) and isn't going to see a high rate of change (so far, you're the only author).

I'm looking into what to do with Travis in the long term - the answer might even be turning it off.

So again, leave the repository-wide setting out of this PR for now - that's a separate concern that should be addressed in a separate PR.

Reply:
Got it.  The reason that Travis set up looking for Ruby is because [Ruby is the default language in Travis](https://github.com/travis-ci/docs-travis-ci-com/issues/862).

We may be growing out of our single mono repo for all languages as we submit code in different languages.  Perhaps we should consider having different repos for different languages if we want to enforce test passing as part of our contributions.  

I'm not sure who has the right permissions, but [turning off Travis for this repo](https://stackoverflow.com/questions/30874853/how-do-you-remove-a-github-repository-for-travis-ci-tests) seems like a reasonable solution.

Either way, I'm happy to remove this travis.yml for this review.